src/Form/GroupSubscribeForm.php

```diff
@@ -234,7 +234,7 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
     /** @var EntityInterface $group */
     $group = $membership->getGroup();
 
-    $message = $membership->isActive() ? $this->t('Your are now subscribed to the group.') : $this->t('Your subscription request was sent.');
+    $message = $membership->isActive() ? $this->t('You are now subscribed to the group.') : $this->t('Your subscription request was sent.');
```

Comment:
```suggestion
    $message = $membership->isActive() ? $this->t('You are now subscribed to the group.') : $this->t('Your subscription request has been sent.');
```

The message has (just) been sent. See https://english.stackexchange.com/questions/73143/simple-past-vs-present-perfect-was-vs-has-been